packages/http-client-csharp/package.json

```diff
@@ -0,0 +1,73 @@
+{
+    "name": "@typespec/http-client-csharp",
+    "version": "0.2.0",
+    "author": "Microsoft Corporation",
+    "description": "The typespec library that can be used to generate C# models from a TypeSpec REST protocol binding",
+    "homepage": "https://github.com/Microsoft/typespec",
+    "readme": "https://github.com/Microsoft/typespec/blob/main/packages/http-client-csharp/readme.md",
+    "license": "MIT",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/Microsoft/typespec.git"
+    },
+    "bugs": {
+        "url": "https://github.com/Microsoft/typespec/issues"
+    },
+    "keywords": [
+        "typespec"
+    ],
+    "type": "module",
+    "main": "dist/src/index.js",
+    "scripts": {
+        "clean": "rimraf ./dist ./temp",
+        "build": "tsc -p .",
+        "watch": "tsc -p . --watch",
+        "lint-typespec-library": "tsp compile . --warn-as-error --import @typespec/library-linter --no-emit",
+        "test": "mocha dist/test/**/*.js",
+        "test-official": "c8 mocha --forbid-only dist/test/**/*.js",
+        "lint": "eslint . --ext .ts --max-warnings=0",
+        "lint:fix": "eslint . --fix --ext .ts",
+        "prettier:fix": "prettier --write --config .prettierrc.json ./src/**/*.ts ./test/**/*.ts",
+        "prettier": "prettier --check --config .prettierrc.json ./src/**/*.ts ./test/**/*.ts"
+    },
+    "files": [
+        "dist/src/**"
+    ],
+    "dependencies": {
+        "json-serialize-refs": "0.1.0-0",
+        "winston": "^3.8.2"
+    },
+    "peerDependencies": {
+        "@azure-tools/typespec-azure-core": ">=0.36.0 <1.0.0",
+        "@azure-tools/typespec-client-generator-core": ">=0.36.0 <1.0.0",
+        "@typespec/compiler": ">=0.50.0 <1.0.0",
+        "@typespec/http": ">=0.50.0 <1.0.0",
+        "@typespec/rest": ">=0.50.0 <1.0.0",
+        "@typespec/versioning": ">=0.50.0 <1.0.0",
+        "@typespec/openapi": ">=0.50.0 <1.0.0"
+    },
+    "devDependencies": {
+        "@azure-tools/typespec-azure-core": "0.40.0",
+        "@azure-tools/typespec-client-generator-core": "0.40.0",
+        "@typespec/compiler": "0.54.0",
+        "@typespec/eslint-config-typespec": "0.54.0",
+        "@typespec/eslint-plugin": "0.54.0",
+        "@typespec/http": "0.54.0",
+        "@typespec/library-linter": "0.54.0",
+        "@typespec/rest": "0.54.0",
+        "@typespec/versioning": "0.54.0",
+        "@typespec/openapi": "0.54.0",
+        "@typespec/json-schema": "0.54.0",
+        "@types/lodash.isequal": "^4.5.6",
+        "@types/mocha": "~9.1.0",
+        "@types/node": "~18.13.0",
+        "@types/prettier": "^2.6.3",
+        "c8": "~7.11.0",
+        "eslint": "^8.12.0",
+        "lodash.isequal": "^4.5.0",
```

Comment:
Should remove this dependency, where it is used you can use `asset.deepStrictEqual` instead of asset.ok + lodash and will give you a better error in test with the diff

Reply:
resolving with follow up item https://github.com/microsoft/typespec/issues/3108